shared/src/commonMain/kotlin/io/projectnewm/login/LoginFieldValidator.kt

```diff
@@ -0,0 +1,28 @@
+package io.projectnewm.login
+
+class LoginFieldValidator {
+    public companion object {
+        public fun validate(email: String, password: String): Boolean {
+            return isValidEmail(email) && isValidPassword(password)
+        }
+
+        private fun isValidEmail(email: CharSequence?): Boolean {
```

Comment:
you switched from non-null String in the `validate` function to nullable CharSequence here and nullable String in `isValidPassword`. Stick with either `CharSequence?` or `String?` all the way through (probably CharSequence? I think)